### PR TITLE
Better Logging

### PR DESCRIPTION
**Pull request by rabenhorst**
I needed better logging because I had some problems with pilight. Maybe you could merge my changes into your code.

### CODE REVIEW

**Comment by knalli** [comment]
@mactr0n  Thank you. Wow, the hound cracked up. :( 

I've just removed the comments, I'll investigate that later.

Anyway: The intention of `this.id` instead of the general `Websocket` in log messages is the plugin will be initiated multiple times (that's why [this comment (still) exists](https://github.com/knalli/homebridge-pilight/blob/06960b168128683b6bc666c36df8ad287b6fb74b/index.js#L53). Also just now I'm not sure anymore why I've chosen a map `this.deviceStates` instead of `this.deviceState`.

**Comment by knalli** [comment]
@mactr0n Ironically, neither the hound or the test have found a more serious issue.. your `index.js` has severe code duplication issue.. perhaps a local merge failed?

**Comment by rabenhorst** [comment]
Really strange. I don’t know what happened. Let me just fix it.

Von: Jan Philipp <notifications@github.com<mailto:notifications@github.com>>
Antworten an: knalli/homebridge-pilight <reply@reply.github.com<mailto:reply@reply.github.com>>
Datum: Donnerstag, 4. Februar 2016 um 22:56
An: knalli/homebridge-pilight <homebridge-pilight@noreply.github.com<mailto:homebridge-pilight@noreply.github.com>>
Cc: mactron <sebastian.rabenhorst@rwth-aachen.de<mailto:sebastian.rabenhorst@rwth-aachen.de>>
Betreff: Re: [homebridge-pilight] Better Logging (#2)
Neu gesendet von: <sebastian.rabenhorst@rwth-aachen.de<mailto:sebastian.rabenhorst@rwth-aachen.de>>

@mactr0nhttps://github.com/mactr0n Ironically, neither the hound or the test have found a more serious issue.. your index.js has severe code duplication issue.. perhaps a local merge failed?

**Comment by rabenhorst** [comment]
It should be fixed now. Should I change the connecting message back to `this.id`?

**Comment by knalli** [comment]
Well, I've to look at it. If it does not match, I will skip the specific lines.

**Comment by knalli** [comment]
Please squash your commits and rebase the new one onto the master. I've applied a configuration for the hound, should not bark so much again.

File: index.js [hunk 1]
```diff
@@ -116,14 +126,13 @@ module.exports = function (homebridge) {
...
 
     getServices() {
-
       // TODO
       let informationService = new homebridge.hap.Service.AccessoryInformation()
         .setCharacteristic(homebridge.hap.Characteristic.Manufacturer, 'Pilight Manufacturer')
         .setCharacteristic(homebridge.hap.Characteristic.Model, 'Pilight Model')
         .setCharacteristic(homebridge.hap.Characteristic.SerialNumber, 'Pilight Serial Number');
 
-      let lightbulbService = new homebridge.hap.Service.Lightbulb();
```
Comment (knalli) [inline index.js:126]: I think `Switch` is much better than `LightBulb` (I was not aware of the other types). But then the rest should be renamed also. I don't want to fool the (code) reader.

**Comment by rabenhorst** [comment]
Ok im busy right now and will do the squash in renaming tomorrow

**Comment by rabenhorst** [comment]
I squashed the commits and renamed `lightbulbService` to `switchService`.

**Comment by knalli** [comment]
Landed as e0166ff and 04ab067. I had to extract the commit for better diff changes, but put the credits in the text.

@mactr0n  Thank you for the work.

Note: I've left the changes with the connection, this is much more better for reading.

**Comment by knalli** [comment]
Go for 1.1.0

**Pull request closed without merging**